### PR TITLE
created new pages for v10 for the api docs.

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

File: Extending/Backoffice-UI-API-Documentation/index-v10.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+versionFrom: 10.0.0
+---
+
+# Backoffice UI API Documentation
+
+A library of API Reference documentation is auto-generated from the comments within the Umbraco Source Code.
+
+## [Backoffice UI](https://apidocs.umbraco.com/v10/ui)
+
+Angular, JavaScript, CSS & Less UI API references for building Umbraco backoffice components. Note that V9 is using minified versions of the following files:
```
Comment (eshanrnh) [inline Extending/Backoffice-UI-API-Documentation/index-v10.md:11]:
```suggestion
Angular, JavaScript, CSS & Less UI API references for building Umbraco backoffice components. V10 is using minified versions of the following files:
```

File: Extending/Backoffice-UI-API-Documentation/index-v10.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- The umbraco.directives
+- The umbraco.services
+- The umbraco.resources
+
+__Note: opens a documentation browser that is different from the documentation section you're viewing now.__
```
Comment (eshanrnh) [inline Extending/Backoffice-UI-API-Documentation/index-v10.md:17]: :::note
Opens a documentation browser that is different from the documentation section you're viewing now.
:::

File: Extending/Backoffice-UI-API-Documentation/index-v10.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+## [UI Examples](https://our.umbraco.com/packages/developer-tools/ui-examples/)
+There is a package available called 'UI Examples'. This package adds a new 'UI Examples' section to your Umbraco instance with working examples of how to customise the backoffice. 
+
+__Note: opens a documentation browser that is different from the documentation section you're viewing now.__
```
Comment (eshanrnh) [inline Extending/Backoffice-UI-API-Documentation/index-v10.md:23]: :::note
Opens a documentation browser that is different from the documentation section you're viewing now.
:::

File: Reference/API-Documentation/index-v10.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+versionFrom: 10.0.0
+meta.Title: "Umbraco API Documentation"
+meta.Description: "Information on Umbraco API Documentation"
+---
+
+# API Documentation
+
+A library of API Reference documentation is auto-generated from the comments within the Umbraco Source Code.
+
+## C# API Documentation
```
Comment (eshanrnh) [inline Reference/API-Documentation/index-v10.md:11]:
```suggestion
## CSharp API Documentation
```

File: Reference/API-Documentation/index-v10.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Angular, JavaScript, CSS & Less UI API references for building Umbraco backoffice components.
+
+- The umbraco.directives
+- The umbraco.services
+- The umbraco.resources
+
+### [Backoffice UI](https://apidocs.umbraco.com/v10/ui)
+
+__Note: opens a documentation browser that is different from the documentation section you're viewing now.__
```
Comment (eshanrnh) [inline Reference/API-Documentation/index-v10.md:35]: Same as above. Add a Note tag instead

**Review by eshanrnh — APPROVED** [review]
Looks good 👏 Just a couple of suggestions.

**Comment by jonat123** [comment]
Merging it in!

**Pull request merged**